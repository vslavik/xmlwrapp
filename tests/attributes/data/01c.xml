<root first="this &amp; that" second="who is on first?"/>
