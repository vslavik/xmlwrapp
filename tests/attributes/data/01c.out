first=this & that
second=who is on first?
